readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://ColumbusDeveloper.github.io/layout_stop-watch/)
```

Comment:
add link to PR description

Reply:
you should add a demo link to PR description

check this screenshot, add link here
<img width="828" alt="image" src="https://github.com/mate-academy/layout_stop-watch/assets/102411130/2aace703-ea61-440c-9118-70e88d282f0b">